apps/condo/pages/property/[id]/report.tsx

```diff
@@ -64,6 +64,7 @@ import type { RowProps } from 'antd'
 const PROPERTY_REPORT_PAGE_ROW_GUTTER: RowProps['gutter'] = [24, 20]
 const PROPERTY_REPORT_PAGE_ROW_TABLE_GUTTER: RowProps['gutter'] = [0, 40]
 const EMPTY_ROW_STYLE: React.CSSProperties = { height: 'calc(100% - 90px)' }
+const DATE_RANGE_PICKER_STYLE: React.CSSProperties = { width: '100%' }
```

Comment:
Is it ok on desktops?

Reply:
Yup. Thats because of default behaviour for DateRangePicker is to fit it's content but not a parent container